slack-java-client-guice/src/main/java/com/hubspot/slack/client/guice/SlackClientModule.java

```diff
@@ -0,0 +1,29 @@
+package com.hubspot.slack.client.guice;
+
+import com.google.inject.AbstractModule;
+import com.hubspot.slack.client.SlackClientFactory;
+import com.hubspot.slack.client.SlackWebClient;
+import com.hubspot.slack.client.http.NioHttpClient;
+import com.hubspot.slack.client.http.NioHttpClientFactory;
+
+public class SlackClientModule extends AbstractModule {
+
+  @Override
+  protected void configure() {
+    bind(SlackWebClient.Factory.class).toInstance(SlackClientFactory.defaultFactory());
+    bind(SlackClientFactory.class).toInstance(SlackClientFactory.defaultFactory());
+
+    bind(NioHttpClientFactory.class).toInstance(NioHttpClientFactory.defaultFactory());
```

Comment:
I think that if this doesn't install the old `SlackClientModule` we might end up with duplicate binding errors when both are installed. Would it be possible to add a test for this case and update this code if there is in fact an error?